client/post-editor/editor-sidebar/header.jsx

```diff
@@ -23,56 +25,74 @@ import DraftsButton from 'post-editor/drafts-button';
 import PostCountsData from 'components/data/post-counts-data';
 import QueryPostTypes from 'components/data/query-post-types';
 
-function EditorSidebarHeader( { typeSlug, type, siteId, showDrafts, toggleDrafts, allPostsUrl, toggleSidebar } ) {
-	const isCustomPostType = ( 'post' !== typeSlug && 'page' !== typeSlug );
-	const className = classnames( 'editor-sidebar__header', {
-		'is-drafts-visible': showDrafts,
-		'is-loading': isCustomPostType && ! type
-	} );
-	const closeLabel = translate( 'Back' );
-	const closeButtonAction = page.back.bind( page, allPostsUrl );
-	const closeButtonUrl = '';
-	const closeButtonAriaLabel = translate( 'Go back' );
+const EditorSidebarHeader = React.createClass( {
+	displayName: 'EditorSidebarHeader',
```

Comment:
You don't need to define `displayName` when you've assigned `React.createClass` object to a named variable, as its name can be derived from the variable declaration.